src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+    width: $width-star;
+    height: $height-star;
+
+    &:not(:last-child) {
+      margin-right: 4px;
```

Comment:
The right margin for the stars is hardcoded to `4px`. If this value is used in multiple places or could be changed in the future, consider storing it in a variable for ease of maintenance.